components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/internal/FeatureManifestInterface.kt

```diff
@@ -41,4 +41,9 @@ interface FeatureManifestInterface<T> {
      * is quite useful for introspection.
      */
     fun getFeature(featureId: String): FeatureHolder<*>?
+
+    /**
+     * Get a list of feature ids where the feature allows co-enrollment.
```

Comment:
Nit: Since this is public API, I think a note saying that this isn't useful for app developers for day-to-day usage of Nimbus.